skinStyles/extensions/StructuredNavigation/ext.structuredNav.NavigationView.less

```diff
@@ -0,0 +1,49 @@
+/*
+ * Citizen
+ *
+ * SkinStyles for StructuredNavigation
+ * Module: ext.structuredNav.NavigationView.styles
+ * Version: REL1_39 55e2ec0
+ *
+ * Date: 2024-09-06
+*/
+
+/* ext.structuredNav.NavigationView.less */
+.mw-structurednav-navigation-container {
+	background-color: var(--color-surface-2);
+	border: 2px solid var(--border-color-base);
+	font-size: var(--font-size-base);
```

Comment:
Use `var( --font-size-small )` instead. It is the equivalent of `12px` in the [original style](https://github.com/wikimedia/mediawiki-extensions-StructuredNavigation/blob/be436f74c8674c1d61142133f7aebf2639c48f3d/resources/styles/ext.structuredNav.NavigationView.less#L6) for the Vector skin.